rest/taskana-rest-spring/src/main/java/pro/taskana/workbasket/rest/WorkbasketController.java

```diff
@@ -184,6 +219,20 @@
     return response;
   }
 
+  /**
+   * This endpoint updates a given Workbasket.
+   *
+   * @title Update a Workbasket
+   * @param workbasketId the id of the Workbasket which should be updated.
+   * @param workbasketRepresentationModel the new Workbasket.
```

Comment:
the workbasket containing the possible modifications to update?

Reply:
Same as with Task: ` the new Workbasket for the requested id.`